cmd/helm/push.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright The Helm Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	"helm.sh/helm/v3/cmd/helm/require"
+	experimental "helm.sh/helm/v3/internal/experimental/action"
+	"helm.sh/helm/v3/pkg/action"
+)
+
+const pushDesc = `
+Upload a package to a registry.
+
+If the --with-prov flag is specified, the chart MUST have an associated
+provenance file, which will also be uploaded.
+`
+
+func newPushCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
+	client := experimental.NewPushWithOpts(experimental.WithPushConfig(cfg))
+
+	cmd := &cobra.Command{
+		Use:               "push [chart] [remote]",
```

Comment:
Since `chart` and `remote` are required arguments, I think [the convention](https://en.wikipedia.org/wiki/Usage_message#Pattern) would be to put them in angled brackets:

```
push <chart> <remote>
```

Reply:
In this case, I am following the convention in other files (such as `cmd/helm/pull.go`). I think it's better to be consistent. Maybe in a follow up PR we can modify this everywhere.